packages/amplify-console-hosting/commands/help.js

```diff
@@ -0,0 +1,8 @@
+async function run(context) {
+  // print out the help message of your plugin
+  context.print.info('help command to be implemented.');
```

Comment:
amplify hosting serve: Opens your deployed site

amplify hosting configure: Set up custom domains, redirects, password protection, and more via the Amplify Console.

amplify publish: Publishes changes to manually deployed apps. For continuous deployment, please push to your git branch to deploy updates.

amplify hosting remove: Remove hosting from you app.